src/main/java/org/itsallcode/openfasttrace/importer/markdown/MarkdownImporter.java

```diff
@@ -38,87 +41,112 @@
-        transition(State.RATIONALE  , State.DEPENDS    , MdPattern.DEPENDS    , this::endRationale                         ),
-        transition(State.RATIONALE  , State.NEEDS      , MdPattern.NEEDS_INT  , () -> {endRationale(); addNeeds();}        ),
-        transition(State.RATIONALE  , State.NEEDS      , MdPattern.NEEDS      , () -> endRationale()                       ),
-        transition(State.RATIONALE  , State.RATIONALE  , MdPattern.EVERYTHING , this::appendRationale                      ),
+        transition(RATIONALE  , SPEC_ITEM  , MdPattern.ID         , () -> {endRationale(); beginItem();}       ),
+        transition(RATIONALE  , TITLE      , MdPattern.TITLE      , () -> {endRationale(); endItem();}         ),
+        transition(RATIONALE  , COMMENT    , MdPattern.COMMENT    , () -> {endRationale(); beginComment();}    ),
+        transition(RATIONALE  , COVERS     , MdPattern.COVERS     , this::endRationale                         ),
+        transition(RATIONALE  , DEPENDS    , MdPattern.DEPENDS    , this::endRationale                         ),
+        transition(RATIONALE  , NEEDS      , MdPattern.NEEDS_INT  , () -> {endRationale(); addNeeds();}        ),
+        transition(RATIONALE  , NEEDS      , MdPattern.NEEDS      , () -> endRationale()                       ),
+        transition(RATIONALE  , RATIONALE  , MdPattern.EVERYTHING , this::appendRationale                      ),
+        transition(RATIONALE  , TAGS       , MdPattern.TAGS       , () -> {}                                   ),
     
-        transition(State.COMMENT    , State.SPEC_ITEM  , MdPattern.ID         , () -> {endComment(); beginItem();}         ),
-        transition(State.COMMENT    , State.TITLE      , MdPattern.TITLE      , () -> {endComment(); endItem();}           ),
-        transition(State.COMMENT    , State.COVERS     , MdPattern.COVERS     , this::endComment                           ),
-        transition(State.COMMENT    , State.DEPENDS    , MdPattern.DEPENDS    , this::endComment                           ),
-        transition(State.COMMENT    , State.NEEDS      , MdPattern.NEEDS_INT  , () -> {endComment(); addNeeds();}          ),
-        transition(State.COMMENT    , State.NEEDS      , MdPattern.NEEDS      , () -> endComment()                         ),
+        transition(COMMENT    , SPEC_ITEM  , MdPattern.ID         , () -> {endComment(); beginItem();}         ),
+        transition(COMMENT    , TITLE      , MdPattern.TITLE      , () -> {endComment(); endItem();}           ),
+        transition(COMMENT    , COVERS     , MdPattern.COVERS     , this::endComment                           ),
+        transition(COMMENT    , DEPENDS    , MdPattern.DEPENDS    , this::endComment                           ),
+        transition(COMMENT    , NEEDS      , MdPattern.NEEDS_INT  , () -> {endComment(); addNeeds();}          ),
+        transition(COMMENT    , NEEDS      , MdPattern.NEEDS      , () -> endComment()                         ),
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)